src/zmq/zmqpublishnotifier.cpp

```diff
@@ -12,10 +12,10 @@
 
 static std::multimap<std::string, CZMQAbstractPublishNotifier*> mapPublishNotifiers;
 
-static const char *MSG_HASHBLOCK = "hashblock";
-static const char *MSG_HASHTX    = "hashtx";
-static const char *MSG_RAWBLOCK  = "rawblock";
-static const char *MSG_RAWTX     = "rawtx";
+static constexpr char *MSG_HASHBLOCK = "hashblock";
```

Comment:
Starting with C++11 this conversion is no longer allowed. Please cast.

Applies also to the three lines below.

Reply:
@DesWurstes I think you might have misunderstood my comment.

My point was that simply replacing `const` with `constexpr` for `MSG_HASHBLOCK` does not give the result you want to achieve in this case :-)

More specifically you don't get a read-only variable.

What you have in `master`:

```
[cling]$ static const char * MSG_HASHBLOCK = "hashblock";
[cling]$ MSG_HASHBLOCK[0] = 'H';
error: read-only variable is not assignable
```

With your suggested change you get:

```
[cling]$ static constexpr char * MSG_HASHBLOCK = "hashblock";
warning: ISO C++11 does not allow conversion from string literal to 'char *const' [-Wwritable-strings]
[cling]$ MSG_HASHBLOCK[0] = 'H';
Segmentation fault
```

What I think you want:

```
[cling]$ static constexpr const char *MSG_HASHBLOCK = "hashblock";
[cling]$ MSG_HASHBLOCK[0] = 'H';
error: read-only variable is not assignable
```